### ISSUE TITLE
Script not working on mobile URL (m.twitch)

### ISSUE THREAD

**Issue by JaMoLpE88** [pixeltris/TwitchAdSolutions#179]
Greetings, I use Firefox on android with ublock origin and the script and it does not work from the m.twitch mobile URL, you have to put the browser in desktop mode to make it work.
Once you put it in desktop mode it works perfectly, but in the mobile URL the player stays black loading in infinity and it doesn't work, is there a possibility that it also works in the m.twitch mobile URL?
It would be very practical and comfortable. Thank you.

**Comment by TsukiZero** [comment]
IMO, use the Twitch App + Blokada 5 if you are on phone.

**Comment by sangohan** [comment]
you could try https://purpletv.aeong.win/ which also works good imho.

**Comment by TsukiZero** [comment]
> you could try https://purpletv.aeong.win/ which also works good imho.

Yeah, nowadays this works better. Freakin' Amazon found out how Blokada works.

**Comment by JaMoLpE88** [comment]
> you could try https://purpletv.aeong.win/ which also works good imho.

I finally used Xtra and it was perfect.

**Comment by pixeltris** [comment]
I've added a note to https://github.com/pixeltris/TwitchAdSolutions/blob/master/issues.md that the scripts don't work on m.twitch.tv. I don't have any plans on working on a script for the mobile site. There are links to mobile solutions in https://github.com/pixeltris/TwitchAdSolutions/blob/master/full-list.md
